README.md

```diff
@@ -230,11 +230,11 @@ Running Tests
 2. cd RIAK_TEST and run commands
 	* make
 5. Download and put `.riak_test.config` in home directory from following link
-	https://gist.github.com/cmeiklejohn/d96d7167e1a8ed4b128d
+	https://gist.github.com/lythq/f979e11749a868fc33dd
```

Comment:
Great.  But, why even bother having it in a gist anymore?  How about just put the file in this repo?